README.md

```diff
@@ -93,6 +93,18 @@ make test
 make performance-test
 ```
 
+### Updating seed data for unit tests
```

Comment:
Might need to add some clarity here... see this comment:  https://github.com/CMSgov/bcda-app/pull/515/files#r456796847

Reply:
Let me know if this comment clarifies the behavior of the dump.pgdata file.